interp/vm.go

```diff
@@ -414,6 +414,16 @@ func (p *interp) execute(code []compiler.Opcode) error {
 			l, r := p.peekPop()
 			p.replaceTop(str(p.toString(l) + p.toString(r)))
 
+		case compiler.ConcatN:
+			numValues := int(code[ip])
+			ip++
+			values := p.popSlice(numValues)
+			parts := make([]string, 0, numValues)
```

Comment:
Probably more efficient to use `strings.Builder` and `WriteString` here. (Though maybe you can test that assumption!)

Reply:
Done!